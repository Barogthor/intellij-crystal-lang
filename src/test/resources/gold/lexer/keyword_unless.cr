unless some_condition
  then_expression
else
  else_expression
end