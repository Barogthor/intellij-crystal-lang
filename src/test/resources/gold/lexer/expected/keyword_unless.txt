identifier ('unless')
whitespace (' ')
identifier ('some_condition')
whitespace ('\n  ')
identifier ('then_expression')
whitespace ('\n')
else ('else')
whitespace ('\n  ')
identifier ('else_expression')
whitespace ('\n')
end ('end')